chall04/hopham.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+
+n = int(input())
+
+arr = []
+for j in range(n):
+    arr.append(list(input()))
+
+for s in range(n - 1):
+    for a, b in enumerate(arr[s]):
+        if b == '.':
+            if arr[s + 1][a] == ' ':
```

Comment:
you can combine those to:
`if b == '.' and arr[s + 1][a] == ' ':`
I feel like it won't damage readability but will make the code a little bit more pythonic